vehicle/mkvi/software/charging/BUILD

```diff
@@ -0,0 +1,34 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+load("//bazel/tools:defs.bzl", "can_api_files")
+load("//bazel:defs.bzl", "cc_firmware")
+
+exports_files(
+    [
+        "charger.yml",
+    ]
+)
```

Comment:
fyi you need to lint your build files in order for this pr to be mergeable - we use `buildifier` as our linter